samples/cpp/benchmark_app/main.cpp

```diff
@@ -28,10 +28,39 @@
 #include "remote_tensors_filling.hpp"
 #include "statistics_report.hpp"
 #include "utils.hpp"
+
+#if defined(_WIN32) || defined(WIN32)
+# include <windows.h>
+#endif
 // clang-format on
 
 static const size_t progressBarDefaultTotalCount = 1000;
 
+std::string get_console_command(int argc, char* argv[]) {
+    std::stringstream args_string;
```

Comment:
Variable name need not contain its storage kind, but rather its meaning or purpose. In this case I'd use `command`.

Reply:
Noted and applied!